data/assets/scene/solarsystem/planets/earth/kernels/ECLIPDATE.tf

```diff
@@ -0,0 +1,56 @@
+Earth Mean Ecliptic and Equinox of Date frame (ECLIPDATE)
```

Comment:
We'll want to move all of these files into an `identifier` based `asset.resource` as well